src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function sortList(roster) {
+  let rosterItems = roster.querySelectorAll('li');
```

Comment:
Consider using array spread syntax to convert the NodeList to an array, like `const rosterItems = [...roster.querySelectorAll('li')];`